cpp/src/arrow/flight/test_util.h

```diff
@@ -42,20 +41,9 @@ namespace flight {
 
 // ----------------------------------------------------------------------
 // Helpers to compare values for equality
-
-inline void AssertEqual(const FlightInfo& expected, const FlightInfo& actual) {
-  ipc::DictionaryMemo expected_memo;
-  ipc::DictionaryMemo actual_memo;
-  ASSERT_OK_AND_ASSIGN(auto ex_schema, expected.GetSchema(&expected_memo));
-  ASSERT_OK_AND_ASSIGN(auto actual_schema, actual.GetSchema(&actual_memo));
-
-  AssertSchemaEqual(*ex_schema, *actual_schema);
-  ASSERT_EQ(expected.total_records(), actual.total_records());
-  ASSERT_EQ(expected.total_bytes(), actual.total_bytes());
-
-  ASSERT_EQ(expected.descriptor(), actual.descriptor());
-  ASSERT_THAT(actual.endpoints(), ::testing::ContainerEq(expected.endpoints()));
-}
+// GH-47659: move definition to test_util.cc to avoid gmock linking errors
```

Comment:
Could you remove this comment?
We don't need this now, right?

Reply:
Yea I can remove this comment. Fixed